plugins/api-docs/src/components/OpenApiDefinitionWidget/OpenApiDefinitionWidget.tsx

```diff
@@ -28,6 +28,7 @@ const LazyOpenApiDefinition = React.lazy(() =>
 /** @public */
 export type OpenApiDefinitionWidgetProps = {
   definition: string;
+  plugins?: string[];
```

Comment:
Tbh I think it's probably nicer to just add these as options? For example `disableTryItOut?: boolean` and `disableAuthorize?: boolean`. That way we don't lock this into a string type and it's a bit more discoverable. What do you think?

Reply:
Yea I see what you mean for discoverability, just thinking it might be easier to add more later should anyone else need it rather than polluting the annotations with many booleans. Personally, I am not tied to either way so if you think it will get more discoverability by making it into a boolean I can change it!

Reply:
Honestly the more I think about it, I think that I am leaning more towards keeping it an array... I can update more of the docs to include this feature for visibility, but I think the `supportedSwaggerPlugins` object identifies in the code what swagger plugins the code supports. 

I see your point on the string rep concern and so maybe the way forward for that would be to use an object instead. One thing I liked about this approach was that all of the swagger plugins were contained in one annotation so there was some functionality separation between other annotations and that each entity could simply list which plugins they wanted to use. I think this logic becomes muddled if the annotations are separated or an object approach is taken. What do you think @Rugvip